### ISSUE TITLE
package.json does not exist at /package.json

### ISSUE THREAD

**Issue by WilsonPhoo** [LedgerHQ/ledgerjs#278]
Does anyone knows how to solve this?

Using electron-react-boilerplate.
Tried to install this 2 but encountered the following error while running:

"@ledgerhq/hw-app-eth": "^4.35.0",
"@ledgerhq/hw-transport-node-hid": "^4.35.0",

```
pre-binding.js:18 Uncaught Error: package.json does not exist at /package.json
    at Object.exports.find (pre-binding.js:18)
    at Object.eval (usb.js:3)
    at eval (usb.js:526)
    at Object../node_modules/usb/usb.js (renderer.dev.dll.js:20650)
    at __webpack_require__ (renderer.dev.dll.js:21)
    at eval (listenDevices.js?:sourcemap:11)
    at Object../node_modules/@ledgerhq/hw-transport-node-hid/lib/listenDevices.js (renderer.dev.dll.js:216)
    at __webpack_require__ (renderer.dev.dll.js:21)
    at eval (TransportNodeHid.js?:sourcemap:21)
    at Object../node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js (renderer.dev.dll.js:190)

  | exports.find | @ | pre-binding.js:18
-- | -- | -- | --
  | (anonymous) | @ | usb.js:3
  | (anonymous) | @ | usb.js:526
  | ./node_modules/usb/usb.js | @ | renderer.dev.dll.js:20650
  | __webpack_require__ | @ | renderer.dev.dll.js:21
  | (anonymous) | @ | listenDevices.js?:sourcemap:11
  | ./node_modules/@ledgerhq/hw-transport-node-hid/lib/listenDevices.js | @ | renderer.dev.dll.js:216
  | __webpack_require__ | @ | renderer.dev.dll.js:21
  | (anonymous) | @ | TransportNodeHid.js?:sourcemap:21
  | ./node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js | @ | renderer.dev.dll.js:190
  | __webpack_require__ | @ | renderer.dev.dll.js:21
  | ./node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js | @ | TransportNodeHid.js …eference renderer:1
  | __webpack_require__ | @ | bootstrap:724
  | fn | @ | bootstrap:101
  | (anonymous) | @ | App.js:15
  | ./app/App.js | @ | renderer.dev.js:1152
  | __webpack_require__ | @ | bootstrap:724
  | fn | @ | bootstrap:101
  | (anonymous) | @ | Routes.js:5
  | ./app/Routes.js | @ | renderer.dev.js:1366
  | __webpack_require__ | @ | bootstrap:724
  | fn | @ | bootstrap:101
  | (anonymous) | @ | Root.js:6
  | ./app/Root.js | @ | renderer.dev.js:1261
  | __webpack_require__ | @ | bootstrap:724
  | fn | @ | bootstrap:101
  | (anonymous) | @ | index.js:4
  | ./app/index.js | @ | renderer.dev.js:34675
  | __webpack_require__ | @ | bootstrap:724
  | fn | @ | bootstrap:101
  | 0 | @ | renderer.dev.js:82557
  | __webpack_require__ | @ | bootstrap:724
  | module.exports../app/App.js.Object.defineProperty.value | @ | bootstrap:791
  | (anonymous)

```

**Comment by itsMikeLowrey** [comment]
I also am having this issue. Im using vue js and electron.

**Comment by itsMikeLowrey** [comment]
It seems to be a issue with the node usb module causing issues. It has caused issues with ledger js [here](https://github.com/LedgerHQ/ledgerjs/issues/242). There is a fix pending [here](https://github.com/tessel/node-usb/issues/290)

**Comment by gre** [comment]
the fix will be to kill the use of `node-usb` in the future 🤞  , we actually have many issues with it, usb events seems unreliable.

**Comment by itsMikeLowrey** [comment]
Will the pending pr on the node-repo fix the issue momentarily? or do I have a misunderstanding of the problem?

**Comment by gre** [comment]
I have no idea. We have never experienced this problem on our side.

**Comment by itsMikeLowrey** [comment]
Heyyyy @WilsonPhoo  and anyone who runs across this in the future. I had this same issue. I looked at the warning labels and it seems to be a common issue with webpack processing the usb node modules and messing up its location. It is detailed in another issue on this repo [here](https://github.com/LedgerHQ/ledgerjs/issues/242). It is not a ledger js issue but a node-usb issue with webpack. My errors looked like this:   
```
warning  in ./node_modules/usb/usb.js
Critical dependency: the request of a dependency is an expression
```
for several usb related dependencies. I simply added a webconfig that ignored the problem modules:
```
    externals: {
      'node-hid': 'commonjs node-hid',
      'usb': 'commonjs usb'
    }
```
This solved this issue for me. Its not a ledger.js issue but I hope this can save anyone in the future who has this issue in the future.
